src/server/index.ts

```diff
@@ -1,3 +1,4 @@
+import './util/tracing'
```

Comment:
we usually put relative imports below the library imports

Reply:
for tracing to works, this has to be the first thing to be import

Reply:
let's add some code comments to document this. otherwise another engineer in the future might just rearrange the imports to match the code style of the rest of the codebase